packages/model-viewer/src/three-components/Renderer.ts

```diff
@@ -86,11 +86,15 @@ export class Renderer extends EventDispatcher {
   constructor(options?: RendererOptions) {
     super();
 
-    const webGlOptions = {alpha: true, antialias: true};
+    const webGlOptions = {
+      alpha: true,
+      antialias: true,
+      powerPreference: 'high-performance' as WebGLPowerPreference
```

Comment:
Is it always the case that we want `'high-performance'`? What are the distinguishing characteristics between the different profiles? Surely there is a trade-off being made here.

Reply:
It only has an effect on dual-GPU systems, telling Chrome to use the faster GPU for this tab. This was the default until Chrome 80, and now it defaults to `low-power`. Based on the feedback I've had from dual-GPU users who were surprised <model-viewer> was not using their good GPU, I think this is a reasonable choice for websites using <model-viewer>.

Reply:
Note, the UA still has authority to override, so user's power preferences still take precedence over this.